python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md

```diff
@@ -29,6 +29,8 @@ The source code for this sample is available [here](https://github.com/microsoft
 * use any sample video with objects as test input to this sample
 * Download the tinyYOLOv2 model from the ONNX Model Zoo
 
+Note: For all the python package dependencies requirements, check 'requirements.txt' file in the sample directory
```

Comment:
Add: You may install these with `pip install -r requirements.txt`.

Reply:
Done